<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reslice3Dto2D</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_image_view">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>570</width>
      <height>570</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(238, 238, 238);
border-width : 2px;
border-style:solid;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_reslice">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>610</y>
      <width>190</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color: rgb(0, 0, 255);
background-color: rgb(155, 202, 255);
border: 0px;
}

QPushButton::pressed
{
color: rgb(0, 0, 255);
background-color: rgb(155, 202, 255);
border: 0px;
}

QPushButton::hover
{
color: rgb(155, 202, 255);
background-color: rgb(0, 0, 255);
border: 0px;
font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>Run Reslice</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_load">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>20</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color: rgb(0, 0, 255);
background-color: rgb(155, 202, 255);
border: 0px;
}

QPushButton::pressed
{
color: rgb(0, 0, 255);
background-color: rgb(155, 202, 255);
border: 0px;
}

QPushButton::hover
{
color: rgb(155, 202, 255);
background-color: rgb(0, 0, 255);
border: 0px;
font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_image_previous">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>630</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color: rgb(0, 0, 255);
background-color: rgb(155, 202, 255);
border: 0px;
}

QPushButton::pressed
{
color: rgb(0, 0, 255);
background-color: rgb(155, 202, 255);
border: 0px;
}

QPushButton::hover
{
color: rgb(155, 202, 255);
background-color: rgb(0, 0, 255);
border: 0px;
font-weight: bold;
}QPushButton
{
color: rgb(0, 0, 255);
background-color: rgb(155, 202, 255);
border: 0px;
}

QPushButton::pressed
{
color: rgb(0, 0, 255);
background-color: rgb(155, 202, 255);
border: 0px;
}

QPushButton::hover
{
color: rgb(155, 202, 255);
background-color: rgb(0, 0, 255);
border: 0px;
font-weight: bold;
}QPushButton
{
color: rgb(0, 0, 255);
background-color: rgb(155, 202, 255);
border: 0px;
}

QPushButton::pressed
{
color: rgb(0, 0, 255);
background-color: rgb(155, 202, 255);
border: 0px;
}

QPushButton::hover
{
color: rgb(155, 202, 255);
background-color: rgb(0, 0, 255);
border: 0px;
font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="dd_series3D">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>50</y>
      <width>491</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(238, 238, 238);
border-width : 2px;
border-style:solid;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_image_next">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>630</y>
      <width>150</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color: rgb(0, 0, 255);
background-color: rgb(155, 202, 255);
border: 0px;
}

QPushButton::pressed
{
color: rgb(0, 0, 255);
background-color: rgb(155, 202, 255);
border: 0px;
}

QPushButton::hover
{
color: rgb(155, 202, 255);
background-color: rgb(0, 0, 255);
border: 0px;
font-weight: bold;
}QPushButton
{
color: rgb(0, 0, 255);
background-color: rgb(155, 202, 255);
border: 0px;
}

QPushButton::pressed
{
color: rgb(0, 0, 255);
background-color: rgb(155, 202, 255);
border: 0px;
}

QPushButton::hover
{
color: rgb(155, 202, 255);
background-color: rgb(0, 0, 255);
border: 0px;
font-weight: bold;
}QPushButton
{
color: rgb(0, 0, 255);
background-color: rgb(155, 202, 255);
border: 0px;
}

QPushButton::pressed
{
color: rgb(0, 0, 255);
background-color: rgb(155, 202, 255);
border: 0px;
}

QPushButton::hover
{
color: rgb(155, 202, 255);
background-color: rgb(0, 0, 255);
border: 0px;
font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>50</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>3D Series Selection</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="txt_load_path">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1081</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(238, 238, 238);
border-width : 2px;
border-style:solid;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_image">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>630</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QListWidget" name="lst_exclude">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>110</y>
      <width>301</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(238, 238, 238);
border-width : 2px;
border-style:solid;</string>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
    <property name="flow">
     <enum>QListView::TopToBottom</enum>
    </property>
    <property name="batchSize">
     <number>100</number>
    </property>
   </widget>
   <widget class="QListWidget" name="lst_include">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>110</y>
      <width>301</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(238, 238, 238);
border-width : 2px;
border-style:solid;</string>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_include">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>300</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color: rgb(0, 0, 255);
background-color: rgb(155, 202, 255);
border: 0px;
}

QPushButton::pressed
{
color: rgb(0, 0, 255);
background-color: rgb(155, 202, 255);
border: 0px;
}

QPushButton::hover
{
color: rgb(155, 202, 255);
background-color: rgb(0, 0, 255);
border: 0px;
font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_include_all">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>110</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color: rgb(0, 0, 255);
background-color: rgb(155, 202, 255);
border: 0px;
}

QPushButton::pressed
{
color: rgb(0, 0, 255);
background-color: rgb(155, 202, 255);
border: 0px;
}

QPushButton::hover
{
color: rgb(155, 202, 255);
background-color: rgb(0, 0, 255);
border: 0px;
font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_exclude_all">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>530</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color: rgb(0, 0, 255);
background-color: rgb(155, 202, 255);
border: 0px;
}

QPushButton::pressed
{
color: rgb(0, 0, 255);
background-color: rgb(155, 202, 255);
border: 0px;
}

QPushButton::hover
{
color: rgb(155, 202, 255);
background-color: rgb(0, 0, 255);
border: 0px;
font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_exclude">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>340</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color: rgb(0, 0, 255);
background-color: rgb(155, 202, 255);
border: 0px;
}

QPushButton::pressed
{
color: rgb(0, 0, 255);
background-color: rgb(155, 202, 255);
border: 0px;
}

QPushButton::hover
{
color: rgb(155, 202, 255);
background-color: rgb(0, 0, 255);
border: 0px;
font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>90</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 238);</string>
    </property>
    <property name="text">
     <string>NOT reslice</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>90</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 238);</string>
    </property>
    <property name="text">
     <string>reslice</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_export_3D">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>570</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>export 3D</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_export_exclude">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>570</y>
      <width>181</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>export not resliced data</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_export_original">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>570</y>
      <width>241</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>export resliced and original data</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>601</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Slice Thickness:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_slice_thickness">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>601</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(238, 238, 238);
border-width : 2px;
border-style:solid;</string>
    </property>
    <property name="inputMask">
     <string>0.00</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_reslice_one">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>570</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>reslice only 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>630</y>
      <width>460</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Slice Thickness: None = same as 2D, 0 = same as 3D, &gt;0 = fixed</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>670</y>
      <width>1281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(207, 207, 207);
font: 75 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Charité Universitätsmedizin Berlin | AG CMR</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
